### ISSUE TITLE
In PHP 7, a "call to undefined function ..." error occurs.

### ISSUE THREAD

**Issue by hansendo** [wckr/wocker#74]
In PHP 7 it was confirmed that mbstring and bz2 could not be found and an error occurred.
For now it's OK to install php7.0-mbstring and php7.0-bz2 with apt-get.
(from google translate.)

**Comment by robdyke** [comment]
I had a similar error 'Call to undefined function xml_parser_create()'.

My fix was to install the missing php7 modules using apt:
```
$ vagrant ssh
wocker ~ $ wocker exec -it wocker bash
root@c9fd5b0bea2f:/var/www/wordpress# apt install php7.0-xml
```

**Comment by akiya64** [comment]
Hi, this is php modules list in Wocker V1.2.
Does anyone know WordPress require php module list.
I am thinking that it would be best WordPressCodex listing php build option.

> [PHP Modules]
> calendar
> Core
> ctype
> curl
> date
> dom
> exif
> fileinfo
> filter
> ftp
> gd
> gettext
> hash
> iconv
> json
> libxml
> mysqli
> mysqlnd
> openssl
> pcntl
> pcre
> PDO
> pdo_mysql
> Phar
> posix
> readline
> Reflection
> session
> shmop
> SimpleXML
> sockets
> SPL
> standard
> sysvmsg
> sysvsem
> sysvshm
> tokenizer
> wddx
> xdebug
> xml
> xmlreader
> xmlwriter
> xsl
> Zend OPcache
> zlib
> 
> [Zend Modules]
> Xdebug
> Zend OPcache

**Comment by ixkaito** [comment]
Fixed this issue on v1.2.1. Thank you all!
